tests/Doctrine/Tests/DBAL/Functional/Schema/SQLServerSchemaManagerTest.php

```diff
@@ -326,13 +328,29 @@ public function testColumnComments(): void
             new Column('commented_type_with_comment', Type::getType('array'), ['comment' => 'Doctrine array type.'])
         );
 
+        // Change column requirements without changing comment.
+        $tableDiff->changedColumns['commented_req_change_column'] = new ColumnDiff(
+            'commented_req_change_column',
+            new Column(
+                'commented_req_change_column',
+                Type::getType('integer'),
+                ['comment' => 'Some comment', 'notnull' => true]
+            ),
+            ['comment', 'notnull'],
+            new Column(
+                'commented_req_change_column',
+                Type::getType('integer'),
+                ['comment' => 'Some comment', 'notnull' => false]
```

Comment:
Since 'comment' is in the list of changed properties, should this one have some different value? Or if it's not changed, should it be removed from the list?

Reply:
"comment" should be removed from changed properties.